config/onprem-iso-config.ign

```diff
@@ -10,7 +10,7 @@
         ],
         "name": "core",
         "sshAuthorizedKeys": [
-          "replace-with-your-ssh-public-key"
+          replace-with-your-ssh-public-key
```

Comment:
There is a build error related to diff on generated files; I think generation in line https://github.com/openshift/assisted-service/blob/b1d97ea3fee06e5b72078c335e50b52d61e44fe7/hack/generate.sh#L71  still transforms
`ssh_authorized_keys` from `onprem-iso-fcc.yaml` into `"replace-with-your-ssh-public-key"` and not expected `replace-with-your-ssh-public-key` in `onprem-iso-config.ign`